server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -241,6 +262,27 @@ export class MongoCollection<T> implements core.ICollection<T>, core.IRetryable
             this.telemetryEnabled, // telemetryEnabled
         );
     }
+
+    private getTelemetryPropertiesFromQuery(queryOrFilter?: any): Map<string, any> | Record<string, any> {
+        const properties: Map<string, any> = new Map();
+        if (!queryOrFilter) {
+            return properties;
+        }
+
+        if (Object.prototype.hasOwnProperty.call(queryOrFilter, "_id")) {
+            properties.set("id", queryOrFilter._id);
+        }
+
+        if (Object.prototype.hasOwnProperty.call(queryOrFilter, "tenantId")) {
```

Comment:
Any reason why we do `Object.prototype.hasOwnProperty.call(queryOrFilter, "tenantId")` instead of `queryOrFilter.tenantId` like below?

Reply:
Some typescript rule asked me to do this....